chat_shell/chat_shell/services/context.py

```
@@ -480,6 +480,58 @@ async def _connect_single_mcp_server(self, server: dict) -> dict:
+                    "success": True,
+                    "tools": tools,
+                    "client": client,
+                    "summary": f"{server_name}({len(tools)})",
+                }
+            else:
+                logger.warning(
+                    "[CHAT_CONTEXT] MCP server %s connected but not ready",
+                    server_name,
+                )
+                return {"success": False, "client": client}
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)